.github/workflows/raedar_tests CI/raedar.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: raedar tests
+
+on:
+  push:
+    branches: [develop ]
```

Comment:
Please change this format to a more acceptable one by the general community.
Kindly use this format while specifying sequences.
```
key:
  - value
  - value
```
However, this is also acceptable and I commend you on the great work.

Reply:
the syntax is now changed